### PR TITLE
add chapter "IPA Client for Testing VPR Server Mode Functionality" to a server_mode/index.rst

### PR DESCRIPTION
**Pull request by w0lek**
The reference to IPAClient repository in order to test vpr server mode

### CODE REVIEW

**Comment by AlexandreSinger** [comment]
@vaughnbetz This LGTM. I think its good to merge.

File: doc/src/vtr/server_mode/index.rst [hunk 1]
```diff
@@ -202,7 +202,17 @@ Draw selected critical path elements example
 
   .. note:: If status is not 1, the field ***DATA*** contains error string.
 
+.. _ipa_client_for_testing_vpr_server_mode_functionality:
 
+IPA Client for Testing VPR Server Mode Functionality
+====================================================
+
+To test VPR server functionality, you can use a special client application called **IPAClient**, which is available in a public Git repository:
```
Comment (vaughnbetz) [inline doc/src/vtr/server_mode/index.rst:210]: I think you should make this first sentence a bit more end-user friendly.

"The interactive path analysis (IPA) client connects to VPR's server mode and allows interactive visualization of timing paths and their placement and routing. This client application is called **IPAClient** and can also be used to test VPR's server mode functionality. It is available in a public Git repository:"

File: doc/src/vtr/server_mode/index.rst [hunk 2]
```diff
@@ -202,7 +202,17 @@ Draw selected critical path elements example
 
   .. note:: If status is not 1, the field ***DATA*** contains error string.
 
+.. _ipa_client_for_testing_vpr_server_mode_functionality:
 
+IPA Client for Testing VPR Server Mode Functionality
```
Comment (vaughnbetz) [inline doc/src/vtr/server_mode/index.rst:207]: I'd change this title to "Interactive Path Analysis Client (IPA)"

File: doc/src/vtr/server_mode/index.rst [hunk 3]
```diff
@@ -202,7 +202,17 @@ Draw selected critical path elements example
 
   .. note:: If status is not 1, the field ***DATA*** contains error string.
 
+.. _ipa_client_for_testing_vpr_server_mode_functionality:
```
Comment (vaughnbetz) [inline doc/src/vtr/server_mode/index.rst:205]: I'd extend this a bit to: "The interactive path analysis (IPA) client is useful for viewing timing paths, and to test VPR's server mode functionality" (with the link embedded somewhere).

**Review by vaughnbetz — CHANGES_REQUESTED** [review]

File: doc/src/vtr/server_mode/index.rst (hunk 2 above)
Comment (w0lek) [reply doc/src/vtr/server_mode/index.rst:207]: resolved ad6430b5797f59b987485ce609d4f0ab74fbdfb8

File: doc/src/vtr/server_mode/index.rst (hunk 1 above)
Comment (w0lek) [reply doc/src/vtr/server_mode/index.rst:210]: resolved, c0662ccce45319c20e9615f470b8eda60ff9b967

File: doc/src/vtr/server_mode/index.rst (hunk 3 above)
Comment (w0lek) [reply doc/src/vtr/server_mode/index.rst:205]: resolved, 041e177e58650dd0196f00d7247ed0d6c9e9591f

**Pull request merged**